src/core/writer.js

```diff
@@ -14,8 +14,14 @@
  */
 /* eslint no-var: error */
 
-import { bytesToString, escapeString } from "../shared/util.js";
+import {
+  bytesToString,
+  escapeString,
+  parseXFAPath,
```

Comment:
This only seems to be used in the core, so instead of putting it in `src/shared/util.js` it should be moved to `src/core/core_utils.js` (and its test).